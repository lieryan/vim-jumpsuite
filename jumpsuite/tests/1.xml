<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
	<testsuite errors="9" failures="0" name="comments.tests.test_api.APITests-20190123100623" skipped="0" tests="9" time="0.117" timestamp="2019-01-23T10:06:28">
		<testcase classname="comments.tests.test_api.APITests" name="test_activate_card" time="0.017" timestamp="2019-01-23T10:06:28">
			<error message="__init__() got an unexpected keyword argument 'logger'" type="TypeError">
<![CDATA[Traceback (most recent call last):
  File "/home/user/Projects/myapp/comments/tests/test_api.py", line 33, in setUp
    self.api = API(logger=logger)
TypeError: __init__() got an unexpected keyword argument 'logger'
]]>			</error>
		</testcase>
	</testsuite>
</testsuites>
