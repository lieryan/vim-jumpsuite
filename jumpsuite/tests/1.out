FAILED. Ran 9 tests in 0.117s (errors=9, failures=0, skipped=0)
/home/user/Projects/myapp/comments/tests/test_api.py:33: setUp : TypeError: __init__() got an unexpected keyword argument 'logger' : self.api = API(logger=logger)
